backend/clubs/views.py

```diff
@@ -4782,6 +4783,29 @@ class TicketViewSet(viewsets.ModelViewSet):
     http_method_names = ["get", "post"]
     lookup_field = "id"
 
+    def _calculate_cart_total(self, cart) -> float:
```

Comment:
Could we make this a free function? `self` isn't used in the body anywhere.

Reply:
Made it a static method.